src/test/java/mattern/william/PhoneBookTest.java

```diff
@@ -0,0 +1,51 @@
+package mattern.william;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by williammattern on 2/7/17.
+ */
+public class PhoneBookTest {
+    PhoneBook phoneBook;
+
+    @Before
+    public void setUp(){
+        phoneBook = new PhoneBook();
+        phoneBook.addEntry("Hank", "123-456-7890");
+
+    }
+    @Test
+    public void lookup() {
```

Comment:
In addition to testing happy path (add an entry, request the entry, confirm it's the right entry), test for unexpected/erroneous behaviors like looking up an entry that doesn't exist or trying to look up entries in an empty phonebook. Treat unit tests as a way of saying to future-you "Here's what should happen when I do this with the thing you're building" and cover as many possible usages as you can think of.